sinks/api/supported_labels.go

```diff
@@ -15,13 +15,15 @@
 package api
 
 const (
-	LabelPodId         = "pod_id"
-	LabelPodName       = "pod_name"
-	LabelPodNamespace  = "pod_namespace"
-	LabelContainerName = "container_name"
-	LabelLabels        = "labels"
-	LabelHostname      = "hostname"
-	LabelResourceID    = "resource_id"
+	LabelPodId               = "pod_id"
+	LabelPodName             = "pod_name"
+	LabelPodNamespace        = "pod_namespace"
+	LabelContainerName       = "container_name"
+	LabelLabels              = "labels"
+	LabelHostname            = "hostname"
+	LabelResourceID          = "resource_id"
+	LabelComputeResourceID   = "compute.googleapis.com/resource_id"
```

Comment:
Ah! Naming is hard :( Lets not overload "resource_id", since it is required for filesystem metrics.
Does "instance id" have to be called "resource_id"? Can we call it "host_id" instead in GCM side?